main.go

```diff
@@ -111,7 +111,7 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	devMode, err := strconv.ParseBool(os.Getenv("DEV_MODE"))
 	if err != nil {
-		devMode = false
+		devMode = true
```

Comment:
this should still default to false be default right?

Reply:
I don't think it's necessary , the devMode = true makes the output human readable as in  (top example): 
https://github.com/openstack-k8s-operators/openstack-operator/pull/360#issuecomment-1620038921

and non dev mode is some Json format which is not easy on the eyes.
So I think for debugging purposes and since we don't have yet a CI logging those logs the devmode is the nicer to have option.